### ISSUE TITLE
Habitat studio build in newer versions (>0.68) results in Powershell Management Module errors

### ISSUE THREAD

**Issue by aumkarpraja** [habitat-sh/habitat#6067]
Hi, so this is related to [this](https://forums.habitat.sh/t/habitat-studio-build-in-newer-versions-results-in-powershell-management-module-errors/971) forum post. Following it up with a Github issue as per @mwrock 's request. Essentially this is happening on a Windows Server 2016 instance with versions of hab 0.68.0 and beyond. 

This issue is not happening on versions before this but it results in a different error where packages building randomly check `C:/Windows/Temp/hab` after artifact creation and fail to find anything there. The hab studio build call is currently run through Jenkins using the Habitat plugin. The error that occurs is below. 

``` 
[***] $ hab studio build habitat-win
Get-Location : The 'Get-Location' command was found in the module 'Microsoft.PowerShell.Management', but the module could not be loaded. For more information, run 'Import-Module Microsoft.PowerShell.Management'.
At C:\hab\pkgs\core\hab-studio\0.68.0\20181108153546\bin\hab-studio.ps1:460 char:22
+   $script:SRC_PATH = Get-Location
+                      ~~~~~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (Get-Location:String) [], ParentContainsErrorRecordException
+ FullyQualifiedErrorId : CouldNotAutoloadMatchingModule
 ```

The only other thing to mention is that hab is installed through Chocolatey.

**Comment by mwrock** [comment]
Thanks for filing this. On the Jenkins node, if you run:

```
hab pkg exec core/powershell/6.1.0 pwsh -command get-location
```

do you get the same error? Also what version of Windows is running on the node?

**Comment by aumkarpraja** [comment]
I don't seem to get that error when running that command, currently on hab version 0.68.0. It's running on Windows Server 2016 Datacenter.

**Comment by mwrock** [comment]
Are you able to do a `hab studio enter` or `hab plan build` outside of Jenkins on the node? It would be interesting if it is something related to the environment provided by Jenkins.

**Comment by aumkarpraja** [comment]
So I managed to prevent the get-location error from happening by running the hab studio build call without the Habitat plugin for Jenkins. The only other issue happening right now is that hab tries to look into the C:/Windows/Temp directory for directories that don't exist. Any ideas to this?

```
â˜… Signed artifact C:\hab\svc\jenkins-slave\data\workspace\workspace\plan-test_devops-pipeline@2\***\***\hab\cache\artifacts\***-***-0.1.0-20190121142004-x86_64-windows.hart.
An error occured in the build!
powershell.exe : Remove-Item : The process cannot access the file 
'C:\Windows\TEMP\hab\pkgs\***\dotnet-core-sdk\2.1.401\20190121141908\bin\shared\Microsoft.NETCore.App\2.1.3\api-ms-win-core-namedpipe-l1-1-0.dll' because it is being used by another 
process.
At C:\hab\svc\jenkins-slave\data\workspace\workspace\plan-test_devops-pipeline@2\***@tmp\durable-bf99c024\powershellWrapper.ps1:3 char:1
+ & powershell -NoProfile -NonInteractive -ExecutionPolicy Bypass -Comm ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Remove-Item : T...nother process.    :String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
At C:\hab\pkgs\core\hab-studio\0.73.0\20190115010701\bin\hab-plan-build.ps1:2012 char:5

+     Remove-Item $tempBase -Recurse -Force

+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

+ CategoryInfo          : WriteError: (C:\Windows\TEMP...pipe-l1-1-0.dll:FileInfo) [Remove-Item], IOException

+ FullyQualifiedErrorId : RemoveFileSystemItemIOError,Microsoft.PowerShell.Commands.RemoveItemCommand
```

**Comment by mwrock** [comment]
The above looks like a genuine bug where the current Windows studio build does not handle concurrent builds gracefully and you could easily get into a situation like this when 2 builds are occurring on the same machine.

I just skimmed over the jenkins plugin code and nothing jumps out at me as to why `Microsoft.PowerShell.Management` would fail to load in the plugin and not outside.

**Comment by aumkarpraja** [comment]
Yup, I can confirm that there does seem to be a bug of some kind with concurrent builds at the moment. Removing the parallel build step stopped other hab builds from wiping out temp files of other package builds going on concurrently.

**Comment by mwrock** [comment]
We fixed the concurrent build issue in 0.75.0. I am closing this for now as the remainder of this issue is likely in the jenkins plugin.
